build.gradle

```diff
@@ -21,5 +21,9 @@ allprojects {
     repositories {
         google()
     }
-}
 
+    ext {
+        KEYSTORE_FILE = rootProject.file('scripts/key.jks')
```

Comment:
This should have been exec/key.jks

Reply:
In the open event repo, there is a folder called exec, inside which all the files were kept. But here we have a folder named scripts, where such files are kept. So I used scripts there.

 Can you please tell what's the use of the above statement.

Reply:
You are saying completely opposite things. open event repo has scripts folder and that's why scripts was used and susi has exec folder. You didn't change anything. You just copied. You should have changed this to exec